### DISCUSSION TITLE
Remove footer on camera entity?

### DISCUSSION THREAD

**Discussion by nvOC-Stubo** [briis/unifiprotect#302] [Q&A] [answered]
Is there a way to remove the footer on the camera entity as seen in HASS? The attached example shows the camera name and the recording status. Since my cameras always record, I would like to remove it and the entire footer. Possible?
![Screenshot 2021-09-22 085416](https://user-images.githubusercontent.com/34128039/134347277-7b3cc144-38a1-47ba-9dc8-937bc72c9df2.jpg)

**Comment by briis** [comment, marked as answer]
That is not done by this Integration it is the Lovelace Card you use that puts this on here. If you use the `Picture Entity` card you can add this to the config of the card:

```yaml
type: picture-entity
entity: camera.5-patio
show_name: false
show_state: false
```
I have not tested this, but I do believe this will remove the name and state from the card.

**Comment by nvOC-Stubo** [comment]
That did the trick. Thank You!
